packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -0,0 +1,426 @@
+  }
+
+  :host(#{$dds-prefix}-filter-panel-modal) {
+    @extend :host(#{$prefix}-modal);
+
+    @include carbon--breakpoint('lg') {
+      display: none;
+    }
+
+    &[open] {
+      @extend :host(#{$prefix}-modal[open]);
+      @include carbon--breakpoint('lg') {
+        display: none;
+      }
+    }
+
+    &[expressive-size='full-width'] {
+      .#{$prefix}--modal-container {
+        max-height: none;
+
+        @include carbon--breakpoint('lg') {
+          width: calc(100% - 2rem);
+          height: calc(100% - 2rem);
+        }
+
+        .#{$prefix}--modal-content {
+          margin: $layout-04 $layout-03;
+          padding: 0;
+        }
+      }
+    }
+
+    .#{$prefix}--modal-container {
+      @include carbon--breakpoint-down('md') {
+        width: 100%;
+        display: grid;
+      }
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-panel) {
+    .#{$prefix}--clear {
+      display: block;
+      &:focus-visible {
+        outline: $spacing-01 solid $focus;
+        margin-bottom: $spacing-01;
+        margin-right: $spacing-01;
+        margin-top: $spacing-01;
+      }
+      &:hover {
+        color: $hover-primary-text;
+      }
+    }
+
+    &:not([has-selections]) {
+      .#{$prefix}--clear {
+        display: none;
+      }
+    }
+    @include carbon--breakpoint-down('lg') {
+      display: none;
+    }
+  }
+
+  :host(#{$dds-prefix}-filter-panel-composite) {
+    .#{$prefix}--filter-button {
+      @include use-carbon-productive-tokens();
+      @include carbon--type-style('body-short-01');
+
+      cursor: pointer;
+      border: 1px solid $interactive-03;
+      color: $interactive-03;
+      padding: $spacing-05;
+      margin-top: $spacing-05;
+      margin-left: $spacing-05;
+      background-color: transparent;
+      width: calc((#{$spacing-10} + #{($spacing-10)}));
+      @include carbon--breakpoint('lg') {
+        display: none;
+      }
+      &:hover {
+        background-color: $hover-tertiary;
+        color: $inverse-01;
+      }
+    }
+  }
+
+  :host(#{$dds-prefix}-checkbox) {
+    padding-left: calc((#{$spacing-04} + #{($spacing-01)}));
+    padding-bottom: calc((#{$spacing-01} + #{($spacing-02)}));
+    @extend :host(#{$prefix}-checkbox);
+    @include use-carbon-productive-tokens();
+    @include carbon--type-style('body-short-01');
+
+    color: $text-02;
+    &:hover {
+      color: $text-01;
+    }
+    &[checked] {
+      color: $text-01;
+    }
+    label.#{$prefix}--checkbox-label {
+      padding-top: $spacing-01;
+      padding-left: $spacing-06;
+    }
+    label.#{$prefix}--checkbox-label::before {
+      border: 1px solid $icon-02;
+    }
+    label.#{$prefix}--checkbox-label:hover::before {
+      border: 1px solid $icon-01;
+    }
+  }
+  :host(#{$dds-prefix}-input-select) {
+    &:hover {
+      background-color: $hover-ui;
+    }
+    .#{$prefix}--input_container {
+      &:focus-visible {
+        outline: $spacing-01 solid $focus;
+      }
+    }
+
+    display: block;
```

Comment:
Removing this broke the `input-select` hover and selected state styling for me

Reply:
Ah, my mistake. It threw me off because it was sitting in the middle of nowhere i thought it was overlooked. Suggest placing it at the start of the block.